### DISCUSSION TITLE
开启插件后 清除浏览器数据后 搜索记录不会被清除

### DISCUSSION THREAD

**Discussion by lyiton** [the1812/Bilibili-Evolved#1381] [提问 / Q&A]
开启插件后 清除浏览器数据后 搜索记录不会被清除 可能是bug

**Comment by the1812** [comment]
搜索记录没记在浏览器里, 你直接在搜索框那里点清除历史就行了

**Comment by lyiton** [comment]
我的浏览器是设置了退出时自动清除浏览数据，既然这样，建议加入关闭浏览器时也即退出插件时能自动清除搜索历史，这样才能保护好隐私




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "the1812/Bilibili-Evolved"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年12月27日(星期天) 晚上10:16
收件人:&nbsp;"the1812/Bilibili-Evolved"<Bilibili-Evolved@noreply.github.com&gt;;
抄送:&nbsp;"422701095"<422701095@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [the1812/Bilibili-Evolved] 开启插件后 清除浏览器数据后 搜索记录不会被清除 (#1381)





 
搜索记录没记在浏览器里, 你直接在搜索框那里点清除历史就行了

**Reply by the1812** [reply]
好吧
